bin/rails

```diff
@@ -4,7 +4,7 @@
 
 ENGINE_ROOT = File.expand_path('../..', __FILE__)
 ENGINE_PATH = File.expand_path('../../lib/hyrax/batch_ingest/engine', __FILE__)
-APP_PATH = File.expand_path('../../test/dummy/config/application', __FILE__)
+#APP_PATH = File.expand_path('../../test/dummy/config/application', __FILE__)
```

Comment:
@cjcolvar why did you comment this out?

Reply:
I commented this out since it was blocking me from running rails generators to make the models.